corehq/toggles.py

```diff
@@ -5,13 +5,43 @@
 import math
 from toggle.shortcuts import toggle_enabled, set_toggle
 
-Tag = namedtuple('Tag', 'name css_class')
-TAG_ONE_OFF = Tag(name='One-Off', css_class='danger')
-TAG_EXPERIMENTAL = Tag(name='Experimental', css_class='warning')
-TAG_PRODUCT_PATH = Tag(name='Product Path', css_class='info')
-TAG_PRODUCT_CORE = Tag(name='Core Product', css_class='success')
-TAG_PREVIEW = Tag(name='Preview', css_class='default')
-TAG_UNKNOWN = Tag(name='Unknown', css_class='default')
+Tag = namedtuple('Tag', 'name css_class description')
+TAG_ONE_OFF = Tag(
+    name='One-Off',
+    css_class='danger',
+    description="This feature flag was created for one specific use-case. "
+    "Please don't enable it for your project without first talking to the tech "
+    "team. This is not fully supported and may break other features.",
+)
```

Comment:
@dimagi/product This is the main addition of this PR.  We can now add descriptions of what each tag means and the appropriate description will appear on each feature flag page.  This should hopefully reduce any ambiguity around what feature flags people should/shouldn't use.

If we decide to merge this, I'd appreciate it if someone from product took a pass at these descriptions.